.github/workflows/ci.yml

```
@@ -89,7 +106,7 @@ jobs:
       - name: Build (coverage)
         if: ${{ matrix.coverage }}
         working-directory: ./dynamic-neural-field-composer
-        run: cmake --build build/linux-debug --parallel $(nproc)
+        run: cmake --build build/linux-debug --target dnf_composer_tests --parallel $(nproc)
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:72: Quote this to prevent word splitting

(shellcheck)